docs/includes/extracts-option-requires.yaml

```diff
@@ -1,43 +1,48 @@
 ---
-ref: option-requires-4.2
+ref: option-requires-version
 content: |
   This option is available since MongoDB {{version}} and will result in an
   exception at execution time if specified for an older server version.
+---
+ref: option-requires-4.2
+source:
+  file: extracts-option-requires.yaml
+  ref: option-requires-version
 replacement:
   version: "4.2"
 ---
 ref: option-requires-4.4
-content: |
-  This option is available since MongoDB {{version}} and will result in an
-  exception at execution time if specified for an older server version.
+source:
+  file: extracts-option-requires.yaml
+  ref: option-requires-version
 replacement:
   version: "4.4"
 ---
 ref: option-requires-5.0
-content: |
-  This option is available since MongoDB {{version}} and will result in an
-  exception at execution time if specified for an older server version.
+source:
+  file: extracts-option-requires.yaml
+  ref: option-requires-version
 replacement:
   version: "5.0"
 ---
 ref: option-requires-5.3
-content: |
-  This option is available since MongoDB {{version}} and will result in an
-  exception at execution time if specified for an older server version.
+source:
+  file: extracts-option-requires.yaml
+  ref: option-requires-version
 replacement:
   version: "5.3"
 ---
 ref: option-requires-6.0
-content: |
-  This option is available since MongoDB {{version}} and will result in an
-  exception at execution time if specified for an older server version.
+source:
+  file: extracts-option-requires.yaml
+  ref: option-requires-version
 replacement:
   version: "6.0"
 ---
 ref: option-requires-7.0
-content: |
-  This option is available since MongoDB {{version}} and will result in an
-  exception at execution time if specified for an older server version.
+source:
+  file: extracts-option-requires.yaml
+  ref: option-requires-version
 replacement:
   version: "7.0"
-...
+...
```

Comment:
Not sure if this is needed, but the original file (and most) have a trailing newline character.